test/workbox-build/node/lib/module-registry.js

```diff
@@ -0,0 +1,55 @@
+/*
+  Copyright 2019 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+const expect = require('chai').expect;
+const proxyquire = require('proxyquire');
+
+describe(`[workbox-build] lib/create-module-imports.js`, function() {
+  const MODULE_PATH = '../../../../packages/workbox-build/src/lib/module-registry';
+  let moduleRegistry;
+
+  beforeEach(() => {
+    const ModuleRegistry = proxyquire(MODULE_PATH, {
+      path: {
+        posix: {
+          resolve: () => '/path/to/node_modules',
+        },
+      },
+    });
+    moduleRegistry = new ModuleRegistry();
+  });
+
+  it(`getImportStatements() should return [] when nothing is used`, function() {
+    expect(moduleRegistry.getImportStatements()).to.be.empty;
+  });
```

Comment:
Can you nest this (and the following blocks) within a `describe()` block?

So like this:

```js
describe('getImportStatements()', function() {
  it(`should return [] when nothing is used`, function() {
    expect(moduleRegistry.getImportStatements()).to.be.empty;
  });
});
```